README.md

```diff
@@ -21,6 +21,38 @@ CPython](https://github.com/python/cpython/blob/e0433c1e7/Doc/tools/rstlint.py).
 - focus on finding errors that are **not** visible to sphinx-build.
 
 
+## Using Sphinx Lint
+
+To use Sphinx Lint, run:
+
+```console
+$ sphinx-lint           # check all dirs and files
+$ sphinx-lint file.rst  # check a single file
+$ sphinx-lint docs      # check a directory
+$ sphinx-lint -i venv   # ignore a file/directory
+$ sphinx-lint -h        # for more options
+```
```

Comment:
```suggestion
```sh
sphinx-lint           # check all dirs and files
sphinx-lint file.rst  # check a single file
sphinx-lint docs      # check a directory
sphinx-lint -i venv   # ignore a file/directory
sphinx-lint -h        # for more options
```
```